### ISSUE TITLE
Chart improvements

### ISSUE THREAD

**Issue by rjahn** [sibvisions/flutter_jvx#203]
Pie chart:

If you have long labels, the labels are inside the chart and overlap other areas in the chart. With short labels it looks better but only as long as the segment is big enough.

Same in chart legend. Label too long, overlaps other labels.

We could try if there's a feature in the chart library to move labels outside the chart and shorten the legend or set clipping areas?

If we can't fix it with existing API: open an issue for the chart library

**Comment by ToniHeiss** [comment]
Example for the pie chart.

![grafik](https://github.com/sibvisions/flutter_jvx/assets/72435900/6bb5d60c-ebd6-4223-875b-4c075b84c298)

**Comment by ToniHeiss** [comment]
Affects new charts implemented by #169

**Comment by ToniHeiss** [comment]
There does not seem to be a solution for the labels in pie charts with this library.

**Comment by ToniHeiss** [comment]
Current open issues:
https://github.com/entronad/graphic/issues/275
https://github.com/entronad/graphic/issues/259

**Comment by rjahn** [comment]
labels outside the chart are working with rose chart, but I guess this chart type is a special solution?

**Comment by ToniHeiss** [comment]
![grafik](https://github.com/sibvisions/flutter_jvx/assets/72435900/7b6c97d8-d0d9-40c4-a1ef-62205ea63d99)
would look like this then

**Comment by rjahn** [comment]
It was just an input that this chart type shows labels outside

**Comment by ToniHeiss** [comment]
Yeah, the chart seems to do that automatically. PieCharts have the label inside them and RoseCharts have them outside.

**Comment by ToniHeiss** [comment]
PieChart selection ticket:
https://github.com/entronad/graphic/issues/276

Currently only possible to select the line in the middle between elements.

**Comment by entronad** [comment]
The pie label issue solution is here:

https://github.com/entronad/graphic/issues/275#issuecomment-2412696892

**Comment by rjahn** [comment]
Some things are clear now, but not all. Maybe we should switch to another library.
